addons/payment/static/src/js/payment_form.js

```diff
@@ -391,12 +418,18 @@ odoo.define('payment.payment_form', function (require) {
             return $(element).data('acquirer-id');
         },
         displayError: function (title, message) {
-            return new Dialog(null, {
-                title: _t('Error: ') + title,
-                size: 'medium',
-                $content: message || "",
-                buttons: [
-                {text: _t('Ok'), close: true}]}).open();
+            var $checkedRadio = this.$('input[type="radio"]:checked'),
```

Comment:
I would also HTML escape the two parameters 'title' and 'message'.
There's an issue in doing that, I see that you are sending errors with HTML. Which will escape them.